README.md

```diff
@@ -4,6 +4,10 @@ ToME is a [rogue-like](https://en.wikipedia.org/wiki/Roguelike) game.
 
 ## Getting Started
 
+### Objective of this fork
+
+A few bugs have been corrected for the code to compile in macOS Big Sur (11.2)
+
```

Comment:
You can omit this change since it wont be a fork once I merge the PR :)

Reply:
OK :)